src/js/_enqueues/lib/link.js

```diff
@@ -181,16 +181,16 @@
 
 			if ( isTouch ) {
 				// Close the onscreen keyboard.
-				inputs.url.focus().blur();
+				inputs.url.trigger( 'focus' ).trigger( 'blur' );
 			} else {
 				/*
 				 * Focus the URL field and highlight its contents.
 				 * If this is moved above the selection changes,
 				 * IE will show a flashing cursor over the dialog.
 				 */
 				window.setTimeout( function() {
-					inputs.url[0].select();
-					inputs.url.focus();
+					inputs.url[0].trigger( 'select' );
```

Comment:
jQuery array notation returns a DOM object rather than a jQuery object, so this is a native JS `trigger()` you are calling: which doesn't exist.

https://stackoverflow.com/questions/13774601/jquery-object-array-notation

Reply:
Good catch, not sure why this didn't cause a failure in my manual test, but might be a scenario I didn't properly hit 🤔

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/1032/commits/a2b88f11d7367468c58350f2126fda700727a928